addon/bin/sign

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+const version = require('../package.json').version;
+const resolve = require('path').resolve;
+const xpiPath = resolve(`dist/testpilot-addon-${version}.xpi`);
```

Comment:
Some more things:

* Looks like the addon moved from `dist/testpilot-addon...` to just `dist/addon.xpi`, so we don't need the version in the string here

* I think the `jpm` version in `package.json` needs an upgrade. The version I have (1.0.3) doesn't have `sign` as a command

Upgrading my local `jpm` and running the command manually gets me this far:
```
$ jpm sign --api-key $AMO_USER --api-secret $AMO_SECRET --xpi dist/addon.xpi
JPM [info] Signing XPI: dist/addon.xpi
JPM [error] Server response: Version already exists. ( status: 409 )
JPM [info] FAIL
```
Do we also need to bump the version every time we want to sign it?